### PR TITLE
Organized schema folder

### PR DESCRIPTION
**Pull request by artmello**
- Renamed folder to "schemas"
- Removed unused schemas (TypeBaseNumber is not used but was not removed since it is a basic schema)
- Removed "String" schema and used existent "BaseTypeString" instead
- Modified "list_path_content" and "list_content_pkg" actors to use a string list that do not contain duplicates
- Removed "StringList" schema and used existent "TypeStringList" instead

### CODE REVIEW

**Comment by artmello** [comment]
Following @bertinatto suggestions, this PR is about cleaning up "schema" folder

**Comment by shaded-enmity** [comment]
Thanks for this @artmello the changes look good, however I'd like to propose dropping `BaseTypeString` and `TypeStringList` as well, and rather adopting the less contrived names everywhere - `String` and `StringList`.

The problem I have with these names is that they're confusing - `TypeStringList`, `BaseTypeString` raises, in my mind, the following questions:

- `...TypeString`? Is the value of the string expected to be a *type* name, like `float`, `int` and whatnot?
- Base ... ? So is this just an **base** type for strings and should I use something more concrete? (like `UtfString` or `AsciiString`?)

The bottom line is, I find `String` and `StringList` nice and straightforward, whereas the *other* names `BaseTypeString` and `TypeStringList` seem to represent a wholly different kind of information.

/cc @bertinatto @mfranczy @vinzenz
